### ISSUE TITLE
Add 'sox' to the get-dependencies.sh script

### ISSUE THREAD

**Issue by vladimirvivien** [claritylab/lucida#30]
After installing all components (with no problems) on Ubuntu 14.x, I get the sox command not found when trying out Kaldi.  Sox did not get installed.  Maybe add `sudo apt-get install sox` to get-dependencies.sh

**Comment by viklas** [comment]
I'm including this in the adjustments being made for the docker build (issue #26 ). There's a number of items to add to the 'get-dependencies' as we're discovering....

**Comment by poshaughnessy** [comment]
Thanks for looking at the Docker build, @viklas. That's how I'm trying to get it running on my Mac - through a Ubuntu Docker container. It's my first time using Docker though, and from a fresh container I've been finding a whole bunch of dependencies that I've needed to install first, like unzip, git and make. I've narrowed down on just trying to get the Question-Answering System working, but I'm still battling through various errors when trying the run scripts.

So I'm looking forward to your docker build - I expect it's going to save me a lot of trouble!
